mergeboard/src/app/components/pullRequest/PullRequestSkeleton.tsx

```diff
@@ -2,32 +2,25 @@
 
 export const PullRequestSkeleton = () => {
   return (
-    <div
```

Comment:
Can you create a separate "PRTableSkeleton" component. Keep this one in case we want to use it later. Rename this one to PRCardSkeleton